services/dynamic-sidecar/Makefile

```diff
@@ -0,0 +1,20 @@
+include ../../scripts/common.Makefile
+include ../../scripts/common-service.Makefile
+
+APP_NAME := $(notdir $(CURDIR))
+
+.DEFAULT_GOAL := help
+
```

Comment:
Inspecting the OpenAPI specs of a service is something very useful. We have these options:

1. look at the openapi.yml file. This is normally in ``api/specs`` folder (for aiohttp-based services) or in the services folder (for fastapi-based services)
2. access ``/dev/doc`` of a running service renders swagger like website

In addition, for 1 we can use a vscode extension that can render it  as a swagger UI 

This PR does not provide option 1. 
My suggestion is to add a makefile recipe to produce an openapi.yml (as done already on other services). 
I adapted it here

```Makefile
.env: .env-devel ## creates .env file from defaults in .env-devel
	$(if $(wildcard $@), \
	@echo "WARNING #####  $< is newer than $@ ####"; diff -uN $@ $<; false;,\
	@echo "WARNING ##### $@ does not exist, cloning $< as $@ ############"; cp $< $@)

.PHONY: openapi.json
openapi.json: .env ## Creates OAS document openapi.json
	# generating openapi specs (OAS) file
	export $(grep -v '^#' $< | xargs -0) && python3 -c "import json; from $(APP_PACKAGE_NAME).main import *; print( json.dumps(app.openapi(), indent=2) )" > $@
	# validates OAS file: $@
	@cd $(CURDIR); \
	$(SCRIPTS_DIR)/openapi-generator-cli.bash validate --input-spec /local/$@
```

Notice that it requires having a ``services/dynamic-sidecar/.env-devel`` in the folder where I put the minimal config
```.env
#
# Environment variables used to configure this service
#- To expose:  export $(grep -v '^#' .env | xargs -0)

# SEE services/api-server/src/simcore_service_api_server/settings.py
LOG_LEVEL=DEBUG


# Enables debug
SC_BOOT_MODE=debug-ptvsd


DYNAMIC_SIDECAR_COMPOSE_NAMESPACE=FOO
DYNAMIC_SIDECAR_DOCKER_COMPOSE_DOWN_TIMEOUT=3

```